### DISCUSSION TITLE
0487. 最大连续1的个数 II

### DISCUSSION THREAD

**Discussion by giscus** [itcharge/AlgoNote#188] [Comments]
# 0487. 最大连续1的个数 II

0487. 最大连续1的个数 II 标签：数组、动态规划、滑动窗口 难度：中等 题目链接 0487. 最大连续1的个数 II - 力扣 题目大意 描述：给定一个二进制数组 nums，可以最多将 1 个 0 翻转为 1。 要求：如果最多可以翻转一个 0，则返回数组中连续 1 的最大个数。 说明： 1 <= nums.length <= 105 nums[...

https://algo.itcharge.cn/Solutions/0400-0499/max-consecutive-ones-ii/

**Comment by ZhuMengliang** [comment]
解题思路里面，第二步和第三步，应该是“窗口内0的个数”吧

**Reply by itcharge** [reply]
感谢指正，已修正
